stable/prometheus-adapter/README.md

```diff
@@ -52,6 +52,7 @@ The following table lists the configurable parameters of the Prometheus Adapter
 | `rules.custom`                  | A list of custom configmap rules                                                | `[]`                                        |
```

Comment:
This feature needs clarification in the `Using the chart` section of the README. Please add a new paragraph explaining that Prometheus Adapter serves three different types of metrics: custom, external and resource metrics.  Provide links to Kubernetes docs for each if they exist. Also indicate which helm property to populate to use each type, maybe with an example (or link to an example).

Also mention how serving resource metrics allows prometheus adapter to provide the same functionality/serve the same API as metrics-server, and thus can't be ran simultaneously in the cluster.